src/dk/aau/cs/ds306e18/tournament/ui/RLBotSettingsTabController.java

```diff
@@ -16,19 +16,20 @@
 
 public class RLBotSettingsTabController {
 
-    @FXML
-    private VBox tabRoot;
-    @FXML
-    private Button configPathBtn;
-    @FXML
-    private TextField configPathTextField;
-    @FXML
-    private CheckBox autoCloseRLBotCheckBox;
+    public static RLBotSettingsTabController instance;
+
+    @FXML private VBox tabRoot;
+    @FXML private Button configPathBtn;
+    @FXML private TextField configPathTextField;
+    @FXML private CheckBox autoCloseRLBotCheckBox;
+
 
     final private FileChooser fileChooser = new FileChooser();
 
     @FXML
     private void initialize() {
+        instance = this;
```

Comment:
Is this not redundant, I don't see references or changes related to this elsewhere.

Reply:
It is referenced in the MainController.